tracing-core/src/metadata.rs

```diff
@@ -273,6 +273,47 @@ impl fmt::Display for Level {
     }
 }
 
+#[derive(Debug)]
+/// Returned if parsing a `Level` fails.
```

Comment:
Extremely minor style nit: typically, we put the RustDoc comments before attributes like `#[derive(Debug)]`; can we swap these here?